README.org

```diff
@@ -141,7 +141,9 @@ Helm and Ivy are also supported.  Note that the =helm= and =ivy= packages are no
 :CUSTOM_ID: TRAMP
 :END:
 
-=magit-todos= attempts to work in remote repositories accessed via TRAMP.  Note that if TRAMP can't find the scanner configured in =magit-todos-scanner=, you may need to use directory-local variables to either add the correct path to =tramp-remote-path= or choose a different scanner.
+By default, =magit-todos= does not automatically scan for to-dos in remote repositories accessed via TRAMP. To enable automatic scanning for to-dos in remote repositories, customize or set the variable ~magit-todos-update-remote~ to t.
+
+Remote repository updates won't work if TRAMP can't find the scanner configured in =magit-todos-scanner=. You may need to use directory-local variables to either add the correct path to =tramp-remote-path= or choose a different scanner.
```

Comment:
I'd like this section to be a bit more concise.  Also, two spaces between sentences please (a nitpick, but one that is expected in Emacs; see documented conventions; remember also that it allows `M-a/e` to work correctly).

Reply:
TIL, thanks!